Update /kubernetes product page (#3836)
pulumi/docs: content/topics/kubernetes.md

    description: |
        Pulumi streamlines Kubernetes cluster configuration, management, and app workload deployments to your clusters.

        With Pulumi you can:

    list:
        - Provision Kubernetes clusters on all major cloud providers.
        - Increase productivity using the full ecosystem of dev tools such as IDE auto-completion, type & error checking, linting, refactoring, and test frameworks to validate Kubernetes clusters, app workloads, or both.
        - Automate deployments with CI/CD integrations for [Spinnaker](https://www.pulumi.com/blog/unlocking-spinnaker-with-pulumi/), [Octopus](https://www.pulumi.com/blog/deploying-with-octopus-and-pulumi/), [GitHub Actions](https://www.pulumi.com/blog/continuous-delivery-to-any-cloud-using-github-actions-and-pulumi/), [GitLab](https://www.pulumi.com/blog/continuous-delivery-with-gitlab-and-pulumi-on-amazon-eks/), [Azure DevOps](https://www.pulumi.com/blog/cd-made-easy-with-pulumi-and-azure-pipelines/) and [more](https://www.pulumi.com/docs/guides/continuous-delivery/).
        - Seamlessly manage cloud resources with the Pulumi Kubernetes Operator.
        - And more...

>     cta: REQUEST MORE INFORMATION

Reviewer: Text size is larger on this CTA than other CTAs below and above.  Can we standardize these @cnunciato

Reviewer 2: Already done, yep.

superpowers:
    - title: Run On Any Cloud
      cta: Learn more

>       ctaUrl: "/docs/get-started"

Reviewer 2: Always use `snake_case` for YAML. I'm correcting these, but just FYI.

      cta: Learn more
      ctaUrl: "/docs/get-started"
      iconType: architecture
      description: |
        YAML and templated DSLs force you to write the same boilerplate code over and over. Using
        Pulumi allows you to codify those patterns and best practices so you can stop reinventing
        the wheel and start inventing the platforms of the future.

    - title: Be Proactive, Not Reactive
      cta: Learn more
      ctaUrl: "/docs/get-started"
      iconType: policy
      description: |
        When you enable Pulumi's Policy as Code feature, you instantly gain the power to prevent
        mistakes from being deployed. Enforce security, compliance, cost controls, and best
        practices using policies defined in real languages.

    - title: Reduce Deployment Anxiety
      cta: Learn more
      ctaUrl: "/docs/get-started"
      iconType: testing
      description: |
        Deploying untested code can lead to some unexpected results. Pulumi lets you take advantage
        of common tools, frameworks, and techniques to unit, integration, and property test your
        infrastructure. Ensure your infrastructure is correct before and after deployment.

> 

Reviewer: add a banner with CTA:
"Learn more about Kubernetes at the Cloud Engineering Summit - October 7 & 8" [CTA Button  - "Register Today"] [CTA Link "https://cloudengineering.heysummit.com/?utm_campaign=summit&utm_source=pulumi.com&utm_medium=web"]

            icon: fa-key

          - title: Pulumi Kubernetes Operator
            description: The Pulumi Kubernetes Operator gives you a Kubernetes controller that deploys cloud infrastructure for you and your team.
            icon: fa-people-carry

    - title: Kubernetes Best Practices with Pulumi Crosswalk
      description: |
        Create, deploy, and manage production-ready infrastructure leveraging hosted Kubernetes offerings such as Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE).
      cta: Learn More
      cta_url: "/docs/guides/crosswalk/kubernetes"
      items:
          - title: Day 2 and Beyond
            description: By using Pulumi Crosswalk for Kubernetes, you can benefit from tried-and-true “Day Two and beyond” integrations and playbooks, improving your infrastructure security, manageability, and cost effectiveness.
            icon: fa-sun

          - title: Accessible Kubernetes

>             description: Through Pulumi's Crosswalk for Kubernetes library extensions, the authorship experience has improved to make theKubernetes API more accessible and approachable to operators and developers of all backgrounds.

Reviewer 3: ```suggestion
            description: Through Pulumi's Crosswalk for Kubernetes library extensions, the authorship experience has improved to make the Kubernetes API more accessible and approachable to operators and developers of all backgrounds.
```

            icon: fa-users

          - title: Focus on Delivering Value
            description: With Pulumi you'll focus more on functionality and business logic of your Kubernetes infrastructure and applications, and less on low-level implementation details.
            icon: fa-chalkboard-teacher

          - title: Avoid Pitfalls

>             description: Discover solutions to the hardest Kubernetes problems to avoid mitigating pitfalls around infrastructure, security, governance, reliability, and maintainability of the cluster, it’s workloads, and underlying resources.

Reviewer 3: ```suggestion
            description: Discover solutions to the hardest Kubernetes problems to avoid mitigating pitfalls around infrastructure, security, governance, reliability, and maintainability of the cluster, its workloads, and underlying resources.
```
